tools/TestedPackageVersionsGenerator/PackageVersionDefinition.cs

```diff
@@ -0,0 +1,28 @@
+// <copyright file="PackageVersionDefinition.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace TestedPackageVersionsGenerator;
+
+internal record PackageVersionDefinition
```

Comment:
It's might be confusing that this file is named `PackageVersionDefinition` and the other one is `PackageVersionDefinitions`. I don't have a rename in mind though, but I thought I would share my first impression

Reply:
Fixed in 698e9eb974bcfa6002fd2eb2e39e985886f379f1

Now we have one file with internal class only.